server/openslides/assignments/migrations/0019_assignmentvote_add_user_token.py

```diff
@@ -0,0 +1,18 @@
+# Generated by jsangmeister on 2021-03-18 16:27
+
```

Comment:
Can you name them `user_token_1` to `user_token_3`, so it is clear, that they have to be seen as one unit.